extensions/gdx-setup/src/com/badlogic/gdx/setup/ExternalExtensionsDialog.java

```diff
@@ -0,0 +1,244 @@
+
+    private ArrayList<ExternalExtension> extensionsSnapshot = new ArrayList<ExternalExtension>();
+    private ArrayList<ExternalExtension> extensions;
+    private HashMap<String, ExtensionRowCheckbox> checkBoxeMap = new HashMap<String,ExtensionRowCheckbox>();
+
+    class ExtensionRowCheckbox extends JCheckBox {
+
+        ExtensionRowCheckbox(String selectName) {
+            super(selectName);
+            setOpaque(false);
+            setBackground(new Color(0, 0, 0));
+            setForeground(new Color(255, 255, 255));
+            setFocusPainted(false);
+        }
+
+    }
+
+    public ExternalExtensionsDialog(ArrayList<ExternalExtension> externalExtensions) {
+
+        extensions = externalExtensions;
+
+        contentPane = new JPanel(new GridBagLayout());
+        setContentPane(contentPane);
+        setModal(true);
+        getRootPane().setDefaultButton(buttonOK);
+
+        uiLayout();
+        uiStyle();
+
+        buttonOK.addActionListener(new ActionListener() {
+            public void actionPerformed(ActionEvent e) {
+                onOK();
+            }
+        });
+        buttonCancel.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                onCancel();
+            }
+        });
+
+        setTitle("Third party external extensions");
+        setSize(400, 240);
+        setLocationRelativeTo(null);
+    }
+
+    public void showDialog() {
+        takeSnapshot();
+        setVisible(true);
+    }
+
+    private void uiLayout() {
+
+        topPanel = new JPanel(new GridBagLayout());
+        topPanel.setBorder(BorderFactory.createEmptyBorder(5, 5, 5, 5));
+
+        content = new JPanel(new GridBagLayout());
+        content.setBorder(BorderFactory.createEmptyBorder(20, 20, 20, 20));
+
+        bottomPanel = new JPanel(new GridBagLayout());
+
+        buttonPanel = new JPanel(new GridBagLayout());
+        buttonPanel.setBorder(BorderFactory.createEmptyBorder(5, 5, 5, 5));
+        buttonOK = new JButton("Save");
+        buttonCancel = new JButton("Cancel");
+        buttonPanel.add(buttonOK, new GridBagConstraints(0, 0, 1, 1, 0, 0, CENTER, HORIZONTAL, new Insets(0, 0, 0, 0), 0, 0));
+        buttonPanel.add(buttonCancel, new GridBagConstraints(1, 0, 1, 1, 0, 0, CENTER, HORIZONTAL, new Insets(0, 0, 0, 0), 0, 0));
+        bottomPanel.add(buttonPanel, new GridBagConstraints(3, 0, 1, 1, 1, 1, SOUTHEAST, NONE, new Insets(0, 0, 0, 0), 0, 0));
+
+        contentPane.add(topPanel, new GridBagConstraints(0, 0, 1, 1, 1, 1, NORTH, BOTH, new Insets(0, 0, 0, 0), 0, 0));
+        contentPane.add(content, new GridBagConstraints(0, 1, 1, 1, 1, 1, NORTH, BOTH, new Insets(0, 0, 0, 0), 0, 0));
+        contentPane.add(bottomPanel, new GridBagConstraints(0, 2, 4, 1, 1, 1, SOUTH, HORIZONTAL, new Insets(0, 0, 0, 0), 0, 0));
+
+        warningNotice = new JLabel("List of third party extensions, not maintained by libGDX team");
```

Comment:
Would be great to have the dialog have links for each extension, but seems a bit complicated for now, maybe just adjust the text a little bit so it will make sense for them, and then some day later, do that other thing. 
Something like: please do not contact libgdx for support with this extensions. (under the main label)